e2e/tests/user/registration-ui.spec.ts

```diff
@@ -72,6 +72,7 @@ describe('User registration tests:', () => {
     await (await createAccountPage.getTermsOfUseCheckbox()).check();
     const agreementPageButton = await createAccountPage.getNextButton();
     await agreementPageButton.clickWithEval();
+    await agreementPageButton.dispose();
```

Comment:
please explain these 2 changes

Reply:
`dispose()`: change the object to unusable explicitly instead implicitly relying on Puppeteer tool to do the same job.

Reply:
ok